.eslintrc

```diff
@@ -8,29 +8,41 @@
 		"es6": true
 	},
 	"rules": {
-		"strict": 0,
-		"camelcase": 0,
-		"curly": 0,
-		"indent": [2, "tab", { "SwitchCase": 1 }],
-		"eol-last": 1,
-		"no-shadow": 0,
-		"no-redeclare": 2,
-		"no-extra-bind": 1,
-		"no-empty": 0,
-		"no-process-exit": 1,
-		"no-underscore-dangle": 0,
-		"no-use-before-define": 0,
-		"no-undef": 2,
-		"no-unused-vars": 0,
-		"consistent-return": 0,
-		"no-inner-declarations": 1,
-		"no-loop-func": 1,
-		"space-before-function-paren": [2, "never"],
-		"space-before-blocks": [2, "always"],
-		"space-before-keywords": [2, "always"],
-		"no-console": 0,
-		"comma-dangle": 0,
-		"no-unexpected-multiline": 2,
-		"valid-jsdoc": 2
+		"strict": "off",
```

Comment:
I suppose this will come later?

Files that really need an exception (e.g. the hot reloading runtime) can always disable it with `/* eslint-disable strict */`.